dspace-api/src/main/java/org/dspace/content/Community.java

```diff
@@ -646,13 +646,29 @@ public Group getAdministrators()
     }
 
     /**
+     * Get the in_archive items in this community. The order is indeterminate.
+     *
+     * @return an iterator over the items in the community.
+     * @throws SQLException
+     */
+    public ItemIterator getItems() throws SQLException {
+        String myQuery ="SELECT item.* FROM item, communities2item WHERE " +
+                "item.item_id=communities2item.item_id AND " +
+                "communities2item.community_id= ? " +
+                "AND item.in_archive='1'";
+        System.out.println(myQuery);
```

Comment:
Also the query doesn't work for Postgres which expects item.in_archive to be compared with true not with 1 (which works for Oracle).